app/presenters/claim_csv_presenter.rb

```diff
@@ -61,15 +72,27 @@ def case_type_name
   end
 
   def scheme
-    if %w[Claim::AdvocateClaim Claim::AdvocateInterimClaim Claim::AdvocateSupplementaryClaim].include? type
+    if AGFS_CLAIM_TYPES.include? type
       'AGFS'
-    elsif %w[Claim::LitigatorClaim Claim::InterimClaim Claim::TransferClaim].include? type
+    elsif LGFS_CLAIM_TYPES.include? type
```

Comment:
https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3284/files#r413031287